iosHyperskillApp/iosHyperskillApp/Sources/Modules/StepQuizParsons/Views/StepQuizParsonsControlsView.swift

```diff
@@ -0,0 +1,89 @@
+import SwiftUI
+
+extension StepQuizParsonsControlsView {
+    struct Appearance {
+        let buttonStyle = OutlineButtonStyle(
+            foregroundColor: Color(ColorPalette.primary),
+            minHeight: 46,
+            maxWidth: nil,
+            borderColor: Color(ColorPalette.primaryAlpha38)
+        )
+    }
+}
+
+struct StepQuizParsonsControlsView: View {
+    private(set) var appearance = Appearance()
+
+    let addTabControlConfiguration: ControlConfiguration
+    let removeTabControlConfiguration: ControlConfiguration
+    let downControlConfiguration: ControlConfiguration
+    let upControlConfiguration: ControlConfiguration
+
+    var body: some View {
+        HStack(spacing: LayoutInsets.smallInset) {
+            Button(
```

Comment:
To be honest, I couldn't use it (such constructor isn't available), the only way I found is using `{ Label(title: StringProtocol, systemImage: String) }` as `label` param. Can you explain me more, please?

Reply:
Sorry, this is a new API since iOS 17

<img width="1840" alt="Screenshot 2023-09-06 at 6 42 22 PM" src="https://github.com/hyperskill/mobile-app/assets/8586063/425d6a96-59c0-45e4-b574-33a3c79d2390">